### ISSUE TITLE
'id attribute value is unique' rules page has duplicate IDs

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1163]
https://act-rules.github.io/rules/3ea0c8

There's an `h3` with `id=examples`
https://act-rules.github.io/rules/3ea0c8#examples

then scroll down a bit to the second set of non-normative examples - that also uses an `h3` with `id=examples` . If you click on the link icon next to the second Examples heading it scrolls back up to the first heading.

If you run the page through the W3 validator it reports:

```
Error: Duplicate ID examples.

From line 137, column 1; to line 137, column 45

</a>.</p>↩<h3 id="examples" style="position:relative;"><a hre

Warning: The first occurrence of ID examples was here.

From line 113, column 1; to line 113, column 45

tree.</p>↩<h3 id="examples" style="position:relative;"><a hre

```

**Comment by WilcoFiers** [comment]
Yikes... @Jym77 can you fix? This is because of the examples heading you added to some definitions.

**Comment by Jym77** [comment]
Yikes indeed…

I will quickfix it.

**Comment by Jym77** [comment]
Closed by #1166.
